concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1178,7 +1178,7 @@ jobs:
               type: docker-image
               source:
                 repository: governmentpaas/bosh-cli-v2
-                tag: c88f3e0b03558c987693fad3f180d9052b77342c
+                tag: 4012d25ceb903b46908a830b8e05773ced1c8f86
```

Comment:
This feels like a good opportunity to use YAML anchors to avoid this duplication following the pattern introduced in https://github.com/alphagov/paas-bootstrap/commit/a2bbe7fdad0ba8bda978bc0500a625cd5a4da3c2

Reply:
Sure. I did the change with `git grep -l c88f3e0b03558c987693fad3f180d9052b77342c  | xargs sed -i s/c88f3e0b03558c987693fad3f180d9052b77342c/4012d25ceb903b46908a830b8e05773ced1c8f86/
` which was quicker, but I will do what you suggest :-D

Reply:
done